charts/camunda-platform-8.7/test/integration/testsuites/vars/files/testsuite-preflight.yaml

```diff
@@ -5,18 +5,34 @@ name: Run preflight checks for Camunda
 # Vars without defaults are passed as a Venom var, e.g. "VENOM_VAR_TEST_CLIENT_SECRET".
 vars:
   releaseName: integration
-  skipTestWebModeler: '{{ .SKIP_TEST_WEBMODELER }}'
+  skipTestWebModeler: "{{ .SKIP_TEST_WEBMODELER }}"
+  # Skip the Elasticsearch test
+  skipes: "{{ .SKIP_TEST_ELASTICSEARCH }}"
```

Comment:
Use a clear env var like `skipTestES` or so.

Reply:
@aabouzaid This env var won't work. I'm not sure what criteria venom uses for env vars. If an env var is not registered by the skip key then it skips the test by default.

Reply:
@hamza-m-masood try to make a PoC to reproduce the issue, I don't see any reason why it shouldn't work.